assets/src/scss/_branding.scss

```diff
@@ -0,0 +1,31 @@
+// Anyone may use the ad server under the terms of the license.
+// However, permission to use the EthicalAds brand, logo, and trademarks,
+// are not conferred with the permission to use the code.
```

Comment:
We should avoid this entirely and just not ship the logos.

Reply:
Moved the logos out